setup.py

```diff
@@ -111,10 +110,7 @@ def run(self):
           packages=find_packages(),
           package_data={'hnn_core': [
               'param/*.json',
-              'gui/*.ipynb',
-              'mod/*',
-              'mod/x86_64/*',
-              'mod/x86_64/.lib/*']},
```

Comment:
I think this is redundant with `MANIFEST.in`

Reply:
Why were these lines added to `setup.py` to begin with then?

Reply:
I think I was desperate: https://github.com/jonescompneurolab/hnn-core/pull/127 :-)